modules/misc/numlock.nix

```diff
@@ -23,6 +23,7 @@ in
 
       Service = {
         Type = "oneshot";
+        RemainAfterExit = "yes";
```

Comment:
Can use
```suggestion
        RemainAfterExit = true;
```
for a slightly cleaner code. Same with xsession below.

Reply:
Done